### PR TITLE
Add instructions for running simulator with other iOS devices to README

### PR DESCRIPTION
**Pull request by iamthomasbishop**
This PR adds simple instructions to the README doc for running the iOS simulator with other devices.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,14 +63,30 @@ yarn android
...
 
 ```
 yarn ios
 ```
 
 which will attempt to open your app in the iOS Simulator if you're on a Mac and have it installed.
 
+### Running on Other iOS Devices
+
+To compile and run the app using a different device, open the target device in Xcode by going to `Hardware > Device`, then use:
```
Comment (etoledom) [inline README.md]: I believe that having the simulator running prior the `yarn` command is not strictly necessary. `yarn ios` will launch it if it's not running already.

File: README.md [hunk 2]
```diff
@@ -63,14 +63,30 @@ yarn android
...
-To compile and run the iOS variant of the app, use:
+To compile and run the iOS variant of the app using the _default_ simulator device, use:
 
 ```
 yarn ios
 ```
 
 which will attempt to open your app in the iOS Simulator if you're on a Mac and have it installed.
 
+### Running on Other iOS Devices
```
Comment (etoledom) [inline README.md]: Nit: Devices sound like the real hardware. Maybe we could say "Other iOS Device Simulators"?

File: README.md (hunk 1 above)
Comment (iamthomasbishop) [reply README.md]: Ah ok, I was having issues with launching others straight-away and this worked, so I assumed this was the "proper" way to do it – but now launching straight-away is working so I'll remove the part specific to opening prior to running the command.

File: README.md (hunk 2 above)
Comment (iamthomasbishop) [reply README.md]: Removed – good catch!

**Pull request merged**